docs/en/operations/access_rights.md

```diff
@@ -1,106 +1,65 @@
+ClickHouse support access control management based on [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control) approach.
 
-Users are recorded in the `users` section. Here is a fragment of the `users.xml` file:
+Elements of ClickHouse access control model:
+- [User account](#user-account-management)
+- [Role](#role-management)
+- [Row Policy](#row-policy-management)
 
-``` xml
-<!-- Users and ACL. -->
-<users>
-    <!-- If the user name is not specified, the 'default' user is used. -->
-    <default>
-        <!-- Password could be specified in plaintext or in SHA256 (in hex format).
+!!! info "Note"
+    Some of the access control functionality is available through the [users.xml](settings/settings_users.md) server configuration files, but we don't recommend using this way of user permissions management. Don't combine these methods of access control management, or it may cause unexpected exceptions.
 
-             If you want to specify password in plaintext (not recommended), place it in 'password' element.
-             Example: <password>qwerty</password>.
-             Password could be empty.
 
-             If you want to specify SHA256, place it in 'password_sha256_hex' element.
-             Example: <password_sha256_hex>65e84be33532fb784c48129675f9eff3a682b27168c0ea744b2cf58ee02337c5</password_sha256_hex>
+## User account {#user-account-management}
 
-             How to generate decent password:
-             Execute: PASSWORD=$(base64 < /dev/urandom | head -c8); echo "$PASSWORD"; echo -n "$PASSWORD" | sha256sum | tr -d '-'
-             In first line will be password and in second - corresponding SHA256.
-        -->
-        <password></password>
+A user account is a configuration that allows to authorize someone in ClickHouse. A user account contains:
 
-        <!-- A list of networks that access is allowed from.
-            Each list item has one of the following forms:
-            <ip> The IP address or subnet mask. For example: 198.51.100.0/24 or 2001:DB8::/32.
-            <host> Host name. For example: example01. A DNS query is made for verification, and all addresses obtained are compared with the address of the customer.
-            <host_regexp> Regular expression for host names. For example, ^example\d\d-\d\d-\d\.host\.ru$
-                To check it, a DNS PTR request is made for the client's address and a regular expression is applied to the result.
-                Then another DNS query is made for the result of the PTR query, and all received address are compared to the client address.
-                We strongly recommend that the regex ends with \.host\.ru$.
+- Identification information.
+- [Privileges](../query_language/grant.md#grant-privileges) that define a scope of queries the user can perform.
+- [Row policies](#row-policy-management) that define filters for rows available for a user.
+- Session settings that apply by default at the user's login.
```

Comment:
User account contains identification information, privileges, allowed hosts, settings and constraints, and also list of granted and default roles.

`Role` contains privileges, settings and constraints, and list of granted roles.

`Row policy` contains filters for one specific table and list of roles and/or users which should use this row policy.

`Quota` contains a set of limits for some durations, and list of roles and/or users which should use this quota.

`Settings profile` contains settings and constraints, and list of roles and/or users which should use this quota.